### PR TITLE
Update iptables.py

### PR DESCRIPTION
**Pull request by rulai-jianfang**
##### SUMMARY
Added example of preventing container to visit aws meta server, you may need to change interface name accordingly, I am using docker network name wildcard
https://blog.appsecco.com/an-ssrf-privileged-aws-keys-and-the-capital-one-breach-4c3c2cded3af

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/system/iptables.py [hunk 1]
```diff
@@ -460,6 +460,19 @@
     limit_burst: 20
     log_prefix: "IPTABLES:INFO: "
     log_level: info
+
+- name: Block container to visit 169.254.169.254
```
Comment (felixfontein) [inline lib/ansible/modules/system/iptables.py:464]: It would be good if you could add more information to this example. Right now, it is hard to understand what it does and especially why it does that without the information you provided in the PR's description.

(Especially the link to https://blog.appsecco.com/an-ssrf-privileged-aws-keys-and-the-capital-one-breach-4c3c2cded3af would be helpful IMO.)

File: lib/ansible/modules/system/iptables.py [hunk 2]
```diff
@@ -460,6 +460,19 @@
...
+  iptables:
+    action: insert
+    comment: prevent container to visit aws meta server
```
Comment (felixfontein) [inline lib/ansible/modules/system/iptables.py:467]:
```suggestion
    comment: prevent container to visit AWS meta server
```

File: lib/ansible/modules/system/iptables.py [hunk 3]
```diff
@@ -460,6 +460,19 @@
...
+    chain: FORWARD
+    protocol: tcp
+    destination_port: 80:443
+    destination: 169.254.169.254
+    in_interface: br-+
```
Comment (felixfontein) [inline lib/ansible/modules/system/iptables.py:472]: This deserves more information. What is `br-+`, and how is that related to containers?

File: lib/ansible/modules/system/iptables.py [hunk 4]
```diff
@@ -460,6 +460,19 @@
...
     log_prefix: "IPTABLES:INFO: "
     log_level: info
+
+- name: Block container to visit 169.254.169.254
+  iptables:
+    action: insert
+    comment: prevent container to visit aws meta server
+    chain: FORWARD
+    protocol: tcp
+    destination_port: 80:443
```
Comment (felixfontein) [inline lib/ansible/modules/system/iptables.py:470]: You first write that you block access to that IP, but then you only block access to a certain range of ports (80 to 443). Is there a reason you don't simply block all ports?

**Review by felixfontein — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
@rulai-jianfang Could you please rebase and address the review comments? Thanks.

needs_info

File: lib/ansible/modules/system/iptables.py (hunk 3 above)
Comment (rulai-jianfang) [reply lib/ansible/modules/system/iptables.py:472]: Sure,
br-+ is docker network names in bridge mode start with "br-", this is the docker network containers are using.

File: lib/ansible/modules/system/iptables.py (hunk 1 above)
Comment (rulai-jianfang) [reply lib/ansible/modules/system/iptables.py:464]: The data breach of bank Wellsfargo is caused by the access to this aws meta data server 169.254.169.254, so we try to block all containers' access to this aws meta data server.

Sorry for delayed reply, just found the github email is spam

File: lib/ansible/modules/system/iptables.py (hunk 4 above)
Comment (rulai-jianfang) [reply lib/ansible/modules/system/iptables.py:470]: From aws doc, it seems only 80/443 port is open on this meta data server, so I blocked 80/443 port, maybe block all ports is better
